### PR TITLE
<Description> : Release candidate for v1.9

### PR DESCRIPTION
**Pull request by Khalian**
<Type> : release

## Checklist

- [ ] New Extension
- [ ] New Test
- [ ] Changed more than one extension, but all changes are related
- [x] Trivial change (doesn't require changelog)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,18 @@
...
 
-## [Unreleased]
+## [Release 1.9]
 
-Add subsequent changes that you make in this section. 
+### Fixed bugs 
+
+1. **Array**
+   - `foreach` is now removed since its already in the std library by *Khalian*
+
```
Comment (DanielAsher) [inline CHANGELOG.md]: add `removeAll` removal.

Comment (Khalian) [inline CHANGELOG.md]: Done.

**Comment by Khalian** [comment]
@piv199 @goktugyil ?

**Comment by DanielAsher** [comment]
removalAll still looks to be on master.

**Comment by Khalian** [comment]
@DanielAsher I am aware of that. When I rebase and merge it, master will replay against it and this commit would be on top of the removeAll commit.

**Comment by DanielAsher** [comment]
@Khalian great, thanks.

**Comment by Khalian** [comment]
"Where depreceated/renamed/modified extensiosn?"

Should I include the commit ID or the PR for the modifications ? This wasnt the case for the CHANGELOG entries for the prior release.

**Comment by Khalian** [comment]
@goktugyil Can I go ahead with the merge ?

**Comment by goktugyil** [comment]
ok

**Comment by DanielAsher** [comment]
@Khalian "When I rebase and merge it, master will replay against it and this commit would be on top of the removeAll commit."
removeAll is still in the codebase. could you link to the commit hash that you reference in your comment.

**Comment by Khalian** [comment]
https://github.com/goktugyil/EZSwiftExtensions/commit/dec0a9771b1e9043b6a352a3112d5cad96455acc

**Comment by DanielAsher** [comment]
@Khalian as I mentioned in #392 (and requested review) `removeAll` is not correctly removed from ArrayExtensions.swift as can be seen here: https://github.com/goktugyil/EZSwiftExtensions/blob/master/Sources/ArrayExtensions.swift#L136

Perhaps I'm confused, if so apologies.

**Pull request merged**